packages/api/tests/lambdas/sf-event-sqs-to-db-records/test-utils.js

```diff
@@ -2,53 +2,45 @@
 
 const test = require('ava');
 const cryptoRandomString = require('crypto-random-string');
-const proxyquire = require('proxyquire');
-const sinon = require('sinon');
 const uuidv4 = require('uuid/v4');
 
-const sandbox = sinon.createSandbox();
-const stubRecordExists = sandbox.stub().resolves(true);
-
 const {
   isPostRDSDeploymentExecution,
-  hasNoAsyncOpOrExists,
-  hasNoParentExecutionOrExists,
+  getMessageAsyncOperationCumulusId,
+  getMessageParentExecutionCumulusId,
   getMessageCollectionCumulusId,
   getMessageProviderCumulusId,
-} = proxyquire('../../../lambdas/sf-event-sqs-to-db-records/utils', {
-  '@cumulus/db': {
-    doesRecordExist: stubRecordExists,
-  },
-});
+} = require('../../../lambdas/sf-event-sqs-to-db-records/utils');
```

Comment:
A very nice side effect of the refactoring is that there is less need for mocking, which is 💯